### PR TITLE
Update docker file to use openshift base image

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+LABEL io.k8s.display-name="openshift-state-metrics" \
+      io.k8s.description="This is a component that exposes metrics about OpenShift objects." \
+      io.openshift.tags="OpenShift" \
+      maintainer="Haoran Wang <haowang@redhat.com>"
 
-ENTRYPOINT ["/openshift-state-metrics", "--port=8080", "--telemetry-port=8081"]
+ARG FROM_DIRECTORY=/go/src/github.com/openshift/openshift-state-metrics
+COPY --from=builder ${FROM_DIRECTORY}/openshift-state-metrics  /usr/bin/openshift-state-metrics
 
-EXPOSE 8080
```
Comment (aditya-konarde) [inline Dockerfile]: Why are the the `EXPOSE` lines being deleted?

Comment (wanghaoran1988) [reply Dockerfile]: I see the kube-state-metrics has this removed https://github.com/openshift/kube-state-metrics/blob/master/Dockerfile, I'd like to align with this repo.

**Pull request merged**